pkgs/applications/networking/flexget/default.nix

```diff
@@ -1,8 +1,34 @@
-{ lib, python3Packages, fetchFromGitHub }:
+{ lib
+, fetchFromGitHub
+, python3
+}:
 
-python3Packages.buildPythonApplication rec {
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+
+      jsonschema = super.jsonschema.overridePythonAttrs (oldAttrs: rec {
+        version = "3.2.0";
```

Comment:
FlexGet is keeping jsonschema<4 just to maintain python 3.6 compatibility https://github.com/Flexget/Flexget/commit/b8a0f674b9276035acb6504e55df4ab544658c0b. 
We are not restricted by that, so we can remove the constrain from requirements.in:
```diff
--- i/pkgs/applications/networking/flexget/default.nix
+++ w/pkgs/applications/networking/flexget/default.nix
@@ -18,7 +18,7 @@ python3Packages.buildPythonApplication rec {
     ln -sf requirements.in requirements.txt
 
     # remove dependency constraints
-    sed 's/==\([0-9]\.\?\)\+//' -i requirements.txt
+    sed 's/[>=]=.*//' -i requirements.txt
 
     # "zxcvbn-python" was renamed to "zxcvbn", and we don't have the former in
     # nixpkgs. See: https://github.com/NixOS/nixpkgs/issues/62110
```